br/backup-and-restore-use-cases.md

```diff
@@ -57,24 +66,27 @@ BR directly sends commands to the TiKV cluster and are not dependent on the TiDB
 
 ## Use cases
 
-This document describes the following four use cases:
+This document describes the following use cases:
 
-* [Back up a single table to a network disk (recommended)](#back-up-a-single-table-to-a-network-disk-recommended)
-* [Restore data from a network disk (recommended)](#restore-data-from-a-network-disk-recommended)
-* [Back up a single table to a local disk](#back-up-a-single-table-to-a-local-disk)
-* [Restore data from a local disk](#restore-data-from-a-local-disk)
+* [Back up a single table to a network disk (recommended in production environment)](#back-up-a-single-table-to-a-network-disk-recommended-in-production-environment)
+* [Restore data from a network disk (recommended in production environment)](#restore-data-from-a-network-disk-recommended-in-production-environment)
+* [Back up a single table to a local disk (recommended in testing environment)](#back-up-a-single-table-to-a-local-disk-recommended-in-testing-environment)
+* [Restore data from a local disk (recommended in testing environment)](#restore-data-from-a-local-disk-recommended-in-testing-environment)
 
 It is recommended that you use a network disk to back up and restore data. This spares you from collecting backup files and greatly improves the backup efficiency especially when the TiKV cluster is in a large scale.
 
-> **Note:**
->
-> Before the backup or restoration operation, you need to do some preparations. See [Preparation for backup](#preparation-for-backup) and [Preparation for restoration](#preparation-for-restoration) for details.
+Before the backup or restoration operations, you need to do some preparations:
+
+- [Preparation for backup](#preparation-for-backup)
+- [Preparation for restoration](#preparation-for-restoration)
 
 ### Preparation for backup
 
-For the detailed usage of the `br backup` command, refer to [BR command-line description](/br/backup-and-restore-tool.md#command-line-description).
+In TiDB v4.0.8 and higher versions, BR supports the self-adaptive Garbage Collection (GC). So to avoid manually configuring GC, you only need to register `backupTS` in `safePoint` in PD and make sure that `safePoint` does not move forward during the backup process.
```

Comment:
```suggestion
In TiDB v4.0.8 and later versions, BR supports the self-adaptive Garbage Collection (GC). So to avoid manually configuring GC, you only need to register `backupTS` in `safePoint` in PD and make sure that `safePoint` does not move forward during the backup process.
```